test/e2e/gateway_single_spoke_test.go

```diff
@@ -293,7 +294,9 @@ var _ = Describe("Gateway single target cluster", func() {
 					By("ensuring the authoritative nameserver resolves the hostname")
 
 					// speed up things by using the authoritative nameserver
-					nameservers, err := net.LookupNS(tconfig.ManagedZone())
+					hname := strings.Replace(tconfig.ManagedZone(), "-", ".", -1)
+
+					nameservers, err := net.LookupNS(hname)
 					Expect(err).ToNot(HaveOccurred())
```

Comment:
@R-Lawton Are any of these changes required?

Reply:
no thats my mistake i was getting mixed up with the env vars